libs/blocks/icon-block/icon-block.css

```diff
@@ -346,6 +346,10 @@
   gap: 16px;
 }
 
+html[dir='rtl'] .icon-block.inline .foreground .text-content {
```

Comment:
L200 uses double quotes, this and the declaration in quote.css use single quotes. I'd suggest we stick to just one of them.

On another note, is the `html` tag selector actually necessary? We did have situations in the past where certain components had their own `dir` attribute, different than that of the `html` tag. So removing the `html` saves a few characters, should have the same effect and it might prove useful for some future use-cases.

Reply:
okay,I have changed it to double quotes.
However i have noticed that all the tags do not have the dir tag set to rtl or ltr, but i still have the need to differentiate and set specific rules for both of the alignments and hence i have used html to differentiate.